tag/nitag.yml

```diff
@@ -0,0 +1,1814 @@
+        401:
+          $ref: '#/responses/Unauthorized'
+        default:
+          $ref: '#/responses/Error'
+  /v2/update-current-values:
+    post:
+      tags:
+        - tags
+      summary: Update the current values of multiple tags (SystemLink Server only)
+      description: Push multiple current value updates to multiple tags in one request.
+      operationId: UpdateTagCurrentValues
+      parameters:
+        - in: body
+          name: updates
+          description: Updates
+          schema:
+            type: array
+            items:
+              $ref: '#/definitions/TagUpdate'
+      responses:
+        200:
+          description: Success
+        400:
+          description: Bad Request
+        401:
+          $ref: '#/responses/Unauthorized'
+        default:
+          $ref: '#/responses/Error'
+  /v2/tags/{path}/update-values:
+    post:
+      tags:
+        - tags
+      summary: Send multiple value updates to a tag
+      description: Push a list of current value update objects to update the value of the tag at the path multiple times. This can be used to send value history in bulk.
+      operationId: update-tag-values
+      responses:
+        200:
+          description: Tag Values Updated
+        400:
+          description: Bad Request
+        401:
+          $ref: '#/responses/Unauthorized'
+        409:
+          description: Conflict
+        default:
+          $ref: '#/responses/Error'
+      parameters:
+        - in: path
+          name: path
+          description: Tag path
+          type: string
+          required: true
+        - in: body
+          name: tagValues
+          description: Timestamped values
+          schema:
+            type: array
+            items:
+              $ref: '#/definitions/TagCurrentValue'
+          required: true
+
+definitions:
+  Error:
+    description: Contains error information
+    type: object
+    properties:
+      name:
+        description: String error code
+        type: string
+      code:
+        description: Numeric error code
+        type: integer
+      resourceType:
+        description: Type of resource associated with the error
+        type: string
+      resourceId:
+        description: Identifier of the resource associated with the error
+        type: string
+      message:
+        description: Complete error message
+        type: string
+      args:
+        description: Positional argument values for the error code
+        type: array
+        items:
+          type: string
+      innerErrors:
+        type: array
+        items:
+          $ref: '#/definitions/Error'
+    example:
+      name: Tag.NoSuchTag
+      code: -251909
+      resourceType:
+      resourceId:
+      message: "No such tag: 'foo'"
+      args: ['foo']
+  TagType:
+    type: string
+    title: Tag Data Type
+    enum: &TYPES [DOUBLE, INT, STRING, BOOLEAN, U_INT64, DATE_TIME]
+  TagWithValue:
+    description: A tag and its current value
```

Comment:
Same question as before about tag v.s. value of tag. Does "tag" mean tag name, tag metadata, or what?

Reply:
In the context of this doc, I've tried to just have "tag" mean tag metadata, so path, properties, keywords, and type. If current and aggregate values are included, I try to call them out separately.

Reply:
Okay. Given that what makes up tag metadata is just those 4 things, I think this is fine.